### PR TITLE
Add ReadTagApp

### PR DESCRIPTION
**Pull request by HugoOliveiraSoares**
Hello!

This pull request implements a new feature that allows reading information from an NFC tag using the PN532 module as a reader.
With this feature it is possible to read: 
- UID
- ATQ
- SAK

Uses Seeed-Studio's [PN532](https://github.com/Seeed-Studio/PN532) libraries and [NDEF](https://github.com/don/NDEF) by Don Coleman that were added to the lib directory

### CODE REVIEW

File: src/applications/nfc/ReadTag/ReadTagRun.h
Comment (marivaaldo) [inline src/applications/nfc/ReadTag/ReadTagRun.h:1]: Please rename to `ReadTagRunScreen.h`

File: src/applications/nfc/ReadTag/ReadTagRun.cpp
Comment (marivaaldo) [inline src/applications/nfc/ReadTag/ReadTagRun.cpp:1]: Please rename to `ReadTagRunScreen.cpp`

File: src/domain/entities/DeviceBase.cpp
Comment (marivaaldo) [inline src/domain/entities/DeviceBase.cpp:1]: Please add `i2cInterface` to `begin` method.

File: src/domain/entities/I2CInterfaceBase.h
Comment (marivaaldo) [inline src/domain/entities/I2CInterfaceBase.h:1]: You can use `I2CInterfaceBase` (using `DSDA_PIN` and `DSCL_PIN`) like `SDCardInterfaceBase` to not needs implement I2CInterface on each device.

File: platformio.ini
Comment (marivaaldo) [inline platformio.ini:1]: I'ts possible to install libs `NDEF, PN532, etc` in this file?

**Review by marivaaldo — CHANGES_REQUESTED** [review]
Please see comments. Thank you so much!

**Comment by HugoOliveiraSoares** [comment]
Hi @marivaaldo, the problems found have been fixed. Thanks for the review.

**Review by marivaaldo — CHANGES_REQUESTED** [review]
Please, you can rebase with `develop` and change PR to merge into `develop` branch? Sorry to not comment this previously and thanks for you great job!

**Comment by HugoOliveiraSoares** [comment]
> Please, you can rebase with `develop` and change PR to merge into `develop` branch? Sorry to not comment this previously and thanks for you great job!

Sure, I will rebase the branch with `develop`

**Review by marivaaldo — APPROVED** [review]

**Pull request merged**